docs/develop/03_workshop/03_overview.md

```diff
@@ -53,3 +72,21 @@ participant B as KILT Blockchain
     C-->>-V: submit presentation
     V->>B: check validity of presentation
 ```
+
+### Example: Requesting a travel visa
+
+To take an example of applying for a travel visa:
+
+1. The Embassy (analogous to the Verifier) asks a traveler (analogous to the Claimer), for a specific document or CType. For example, it could be a bank statement. The Embassy asks, "Provide proof of financial stability, and we'll grant you a visa." The traveller gets the bank statement from their bank, gets it attested by the bank (The Attester), and prepares the document.
+2. The document is ready, but the Embassy doesn't need all the information in the document. The embassy wants to know if a traveller has sufficient funds, but they don't need to know any transaction details. So, the traveller redacts or hides these details while presenting.
+3. The traveller presents the document to the embassy.
+4. The embassy verified the document's authenticity by comparing its hash value with the one on their internal system or a decentralized ledger.
+5. Since they trust the Attester (in this case, the bank that attested the bank statement), they approved the visa application.
+
+:::tip Summary
+
+As you can see, the Embassy didn't need to trust the Claimer directly in this system. They trusted the Attester, whom they had previously worked with or respected due to their position. And with that trust, they granted the visa.
```

Comment:
```suggestion
As you can see, the Embassy didn't need to trust the Claimer directly in this system.
They trusted the Attester, whom they had previously worked with or respected due to their position.
And with that trust, they granted the visa.
```

Reply:
Not sure but would be interesting to note that the attester/issuer doesn't know that the holder/claimer has used the credential in anyway.